README.md

```diff
@@ -6,33 +6,66 @@
-* Are language agnostic
+The goal of Habitat is to allow you to automate your application behavior when you create your application, and then bundle your application with the automation it needs to behave with the correct run time behavior, update strategies, failure handling strategies, and scaling behavior, wherever you choose to deploy it.
 
-To learn more about Habitat, please visit the [Habitat website](https://www.habitat.sh).
+## Overview and Demo
+[![Build, Deploy and Manage with Habitat (5:33)](images/overview-youtube-image.jpg)](http://www.youtube.com/watch?v=VW1DwDezlqM)
+
+## Habitat Infographics
+### Where Habitat fits
+
+[![Habitat Flow Infographic](images/habitat-flow-infographic.png)](http://habitat.sh)
+
+[//]: # (Try the interactive infographics on the [website](http://habitat.sh)
+
+### Habitat Workflow
+* [Habitat **Initial Package Build** Flow](https://www.habitat.sh/images/infographics/habitat-initial-package-build-flow-43ccb287.png)
+* [Habitat **Application Rebuild** Flow](https://www.habitat.sh/images/infographics/habitat-application-rebuild-flow-10c48cfb.png)
+* [Habitat **Dependency Update** Flow](https://www.habitat.sh/images/infographics/habitat-dependency-update-flow-f4895875.png
+)
+
+### Habitat Concept
+* [Habitat Architecture Overview](https://www.habitat.sh/images/infographics/habitat-architecture-overview-7e40c4b3.png)
+* [Habitat **Builder** Architecture](https://www.habitat.sh/images/infographics/habitat-builder-architecture-059a4ddd.png
+)
+* [Habitat Promote Packages Through **Channels**](https://www.habitat.sh/images/infographics/habitat-promote-packages-through-channels-06506fd0.png)
+* [Habitat Runtime **Services Group** Binding](https://www.habitat.sh/images/infographics/habitat-runtime-service-group-binding-0e134bf3.png)
+
+### Habitat Integration
+
+Habitat and **Docker**
+* [Initial Docker Container Publishing Flow](https://www.habitat.sh/images/infographics/habitat-initial-docker-container-publishing-flow-1f2ba489.png)
+* [Automated Docker Container Publishing Flow](https://www.habitat.sh/images/infographics/habitat-automated-docker-container-publishing-flow-3cdf9f91.png
+)
+
+Habitat and **Kubernetes**
+* [Three Tiers of Service Deployment](https://www.habitat.sh/images/infographics/habitat-and-kubernetes-three-tiers-of-service-deployment-8c85b4ca.png)
+* [Deploy Services to Kubernetes with Habitat](https://www.habitat.sh/images/infographics/deploy-services-to-kubernetes-with-habitat-flow-567c4ce9.png)
+
+*View all infographics in [docs](https://www.habitat.sh/docs/reference/#infographics)*
+
+## Hands-on Demos
+* [Habitat Packaging System Demo (15 mins)](https://www.habitat.sh/demo/packaging-system/steps/1/)
+* [Habitat Build System Demo (15 mins)](https://www.habitat.sh/demo/packaging-system/steps/1/)
+* [Habitat Process Supervisor Demo (15 mins)](https://www.habitat.sh/demo/packaging-system/steps/1/)
+
+*View all demos and tutorials in [learn page](https://www.habitat.sh/learn/)*
```

Comment:
This might just be me, but all these links up at the top of the readme feels a little overwhelming to me; if I were new to the project, I wouldn't know where to begin. Do we need all of them here, or would a few select/representative ones be more effective as introductions to further exploration on the site?